Fix install, upgrade, deploy, and manage issues (#8638)
pingcap/docs: faq/upgrade-faq.md

### What are the effects of rolling updates?

> When you apply rolling updates to the TiDB services, the running application is affected to some extent. Therefore, it is not recommended that you perform a rolling update during business peaks. You need to configure the minimum cluster topology (TiDB \* 2, PD \* 3, TiKV \* 3). If the Pump or Drainer service is involved in the cluster, it is recommended to stop Drainer before rolling updates. When you upgrade TiDB, Pump is also upgraded.

Reviewer: ```suggestion
When you apply rolling updates to the TiDB services, the running application is affected to varying degrees. Therefore, it is not recommended that you perform a rolling update during business peak hours. You need to configure the minimum cluster topology (TiDB \* 2, PD \* 3, TiKV \* 3). If the Pump or Drainer service is involved in the cluster, it is recommended to stop Drainer before rolling updates. When you upgrade TiDB, Pump is also upgraded.
```

### How to upgrade TiDB using the binary?

> It is not recommended to upgrade TiDB using a binary. Instead, you are advised to [Upgrade TiDB using TiUP](/upgrade-tidb-using-tiup.md), which ensures both version consistency and compatibility.

Reviewer 2: ```suggestion
It is not recommended to upgrade TiDB using a binary. Instead, you are advised to [Upgrade TiDB using TiUP](/upgrade-tidb-using-tiup.md) or TiDB Operator for Kubernetes, which ensures both version consistency and compatibility.
```

> It is not recommended to upgrade TiDB using a binary. Instead, you are advised to [Upgrade TiDB using TiUP](/upgrade-tidb-using-tiup.md) or [Upgrade a TiDB Cluster in Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/upgrade-a-tidb-cluster), which ensures both version consistency and compatibility.

Reviewer: ```suggestion
It is not recommended to upgrade TiDB using the ebinary. Instead, it is recommended to [upgrade TiDB using TiUP](/upgrade-tidb-using-tiup.md) or [upgrade a TiDB cluster in Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/upgrade-a-tidb-cluster), which ensures both version consistency and compatibility.
```
